group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorServiceTest.java

```diff
@@ -139,7 +138,7 @@ public void testStartupShutdown() throws Exception {
     }
 
     @Test
-    public void testConsumerGroupHeartbeatWhenNotStarted() {
+    public void testConsumerGroupHeartbeatWhenNotStarted() throws ExecutionException, InterruptedException {
```

Comment:
Is it worth adding a similar test for each API? That would ensure that we keep the contract.